ee/debug/checkups/init_logs_linux.go

```diff
@@ -3,8 +3,24 @@ package checkups
 import (
 	"archive/zip"
 	"context"
+	"fmt"
+
+	"github.com/kolide/launcher/ee/allowedcmd"
 )
 
-func writeInitLogs(_ context.Context, _ *zip.Writer) error {
-	return nil
+func writeInitLogs(ctx context.Context, logZip *zip.Writer) error {
+	cmd, err := allowedcmd.Journalctl(ctx, "-u", "launcher.kolide-k2.service", "-o", "json")
```

Comment:
putting these in JSON formate, both for this and for windows, produce very verbose logs due to the extra properties it adds associated with each message ... do we want this?

Reply:
The extra info doesn't seem super useful (except for `_PID` maybe, but that should be available in a different checkup anyway) -- I think I'd lean toward not including it just to avoid the extra noise?

Reply:
My inkling is not. I think of this as being pretty simple to a quick "tail the log file". But maybe I'll regret saying that

Reply:
droped the json conversation and just let it print the default on linux. We could add some output options to make it prettier or more customized, but those options might not be present on all distros